periodic/periodic.go

```diff
@@ -546,6 +556,16 @@ func formatDate(d *time.Time) string {
 		d.Hour(), d.Minute(), d.Second())
 }
 
+// Return a jitter time that is (+/-)10% of the duration t
+func addJitter(t time.Duration) time.Duration {
+	if t <= 0 {
+		return time.Duration(0)
```

Comment:
return the original value rather than 0 otherwise the stats will not be correct (falling behind detection when too large % of request are falling behind)

Reply:
this function returns a jitter value that gets added to the original delay. `sleepDuration += addJitter(sleepDuration)`

Reply:
oh right, duh me! (I suppose it could be also doing the "add" - maybe change the name to "calcJitter" or just "jitter()"

Reply:
purist go folks (which I'm not but went through google code reviews before...) tend to frown on functions called getFoo() in favor of just Foo() but it's fine for a non public method anyway